command/ffprobe/README_zh.md

```diff
@@ -0,0 +1,4 @@
+## ffprobe使用教程 
+
+### ffprobe显示视频帧数据
+

```

Comment:
what i mean is you don't need to translate `ffprobe -v error -show_entries stream=avg_frame_rate,r_frame_rate,nb_frames out.mp4` so you can just put it in the origin place.